reference/import_export/product-export.rst

```diff
@@ -16,20 +16,41 @@ The product export is defined in ``src/Pim/Bundle/ConnectorBundle/Resources/conf
 
```

Comment:
and maybe explain that now you can export natively in CSV and XLSX

and that you can finely select which products to export